### ISSUE TITLE
Implementing the script "run_experiment.py" prints the "error" message for every image

### ISSUE THREAD

**Issue by ZainabOuardirhi** [YihongSun/Bayesian-Amodal#1]

**Comment by ZainabOuardirhi** [comment]
Greeting,

First of all thank you for sharing this project.
I would like to ask you if you may have any idea about the reason why the **run_experiment** script prints the error messages, I tried to look if it was because of the libraries versions, but I personally don't think so...

![Screenshot 2022-05-12 104034](https://user-images.githubusercontent.com/81168343/168029996-f656e738-ec3a-48fe-b32d-42d72a0f499f.png)


When I went to check the source code, i find that the error is comming from the **eval_performance** function, specifically from the following part 

![Screenshot 2022-05-12 104405](https://user-images.githubusercontent.com/81168343/168030215-f371eb83-c750-4fdd-b254-e9d2afeb2f13.png)

Thanks in advance!

**Comment by YihongSun** [comment]
Hello,

Since the model's forward pass is wrapped in a try block, the actual error that caused the exception handling is suppressed. 

Try to run the following code outside of the try block to see what the error message is.
```
with torch.no_grad():
    pred_scores, ... = net(org_x=input_tensor...)
```

Please let me know if there is anything else that I can help with!

**Comment by ZainabOuardirhi** [comment]
It worked! Thank you @YihongSun . 
I also tried to train the model using the train_CompNet script. Using the value 1 as the table number it gave me the following error : 
![Screenshot 2022-05-13 151232](https://user-images.githubusercontent.com/81168343/168291942-85a5bd58-dc03-48f2-9b57-dd596f01d059.png)

Using the value 2 it gave me the following error : 
![Screenshot 2022-05-13 151417](https://user-images.githubusercontent.com/81168343/168291964-80c5fc37-b6f2-4aab-99af-569162ee3abf.png)

Are these errors due to lack of data?

Thanks again.

**Comment by DonaldKam** [comment]
I am encountering the same problem. It has not yet been addressed.

**Comment by YihongSun** [comment]
Sorry for the delay response.

For the error in `TABLE_NUM = 1`, it seems that there is an error loading the model that was catched by the try block. The error occurred on [this line](https://github.com/YihongSun/Bayesian-Amodal/blob/4ce9cbd03c7a5c713eb93c051efffefaa0db2ee5/Code/model.py#L141).
Without properly loading the model, `model.fg_models[i]` stayed unchanged as `None`.

For the error in `TABLE_NUM = 2`, the models were successfully loaded (shown in `Dim Reduction - cyclist: (21, 25) --> (19,21)`), but there is an error with the keyword `gt_alignment`. This is a version issue with `Net.py` and to resolve this, simply remove the parameter `gt_alignment=input_alighnments`.

Sorry for the inconvenience and hope it helps!
